common/libs/wallet/myetherwallet.js

```diff
@@ -0,0 +1,353 @@
+import {
```

Comment:
Useful links for this later:

https://github.com/kvhnuke/etherwallet/blob/mercury/app/tests/index.html

https://github.com/kvhnuke/etherwallet/blob/mercury/app/scripts/myetherwallet.js

This file is mostly all translated ES5 -> ES6 from https://github.com/kvhnuke/etherwallet/blob/mercury/app/scripts/myetherwallet.js

`getBlob` is the only new method that wasn't in v3.

Very tentative and needs discussion about strucutre / interaction with PrivKeyWallet